### PR TITLE
Information on speaker assistance and bursaries for the 2016 Summits

### PR DESCRIPTION
**Pull request by davegurnell**
To my understanding, we're offering several support services around the Summits in the spirit of diversity:

- speaker assistance
- bursaries for speakers
- bursaries for attendees

Hopefully the attached blog post and paragraph for the Philadelphia Summit page describe these things effectively. Feedback, suggestions, and corrections welcome!

### CODE REVIEW

File: _posts/2016-01-14-summit_assistance.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+We will assign a mentor to each participating speaker. Your mentor will be available to act as a sounding board as your talk develops from an outline to a fully formed presentation.
+
+### Sound Interesting?
+
+If you are planning on attending or speaking in Philadelphia or Oslo, and you would like to apply for any of the services above, please please fill out the application form linked at the top of the page.
+
+[code-of-conduct]: /conduct.html
+[summits]: /blog/2015/12/11/announcement_summit.html
+[philadelphia]: /event/typelevel-summit-philadelphia/
```
Comment (larsrh) [inline _posts/2016-01-14-summit_assistance.md]: I changed the URLs for the summits:

/event/2016-03-summit-philadelphia/
/event/2016-05-summit-oslo/

**Comment by larsrh** [comment]
@davegurnell Thanks, the text looks good. :+1: from my side.

@milessabin Could you please give this a once-over, too?

File: _posts/2016-01-14-summit_assistance.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+title: "Assistance and Bursaries for the Typelevel Summits"
+
+meta:
+  nav: blog
+  author: davegurnell
+---
+
+As it says in our [code of conduct][code-of-conduct], we are dedicated to creating a harrassment-free, inclusive community of developers. We want to extend this philosophy to the Typelevel Summits in [Philadelphia][philadelphia] and [Oslo][oslo] by providing assistance and bursaries to help speakers and attendees who would otherwise not be able to join us.
+
+<a class="btn large" href="https://docs.google.com/a/underscoreconsulting.com/forms/d/1hhia7etHm_UT4WnQS7JTyGE03z-2-T1xJGujOkvacjs/viewform">Apply for bursaries / speaker assistance</a>
```
Comment (larsrh) [inline _posts/2016-01-14-summit_assistance.md]: For some reason I can't view this form. Is it not public yet? The message I'm getting is:

`This form can only be viewed by users in the owner's organization.`

Comment (davegurnell) [inline _posts/2016-01-14-summit_assistance.md]: Yup -- that's my bad. Should be open access now.

Comment (non) [inline _posts/2016-01-14-summit_assistance.md]: I can confirm that I can access the form.

**Comment by non** [comment]
Looks good to me. :+1:

**Comment by larsrh** [comment]
Thanks much!

**Pull request merged**